libsupport/include/katana/EntityTypeManager.h

```diff
@@ -78,6 +83,171 @@ class EntityTypeManager {
         atomic_entity_type_id_to_entity_type_ids_(
             std::move(atomic_entity_type_id_to_entity_type_ids)) {}
 
+  template <typename ArrowType>
+  struct PropertyColumn {
+    int field_index;
+    std::shared_ptr<ArrowType> array;
+
+    PropertyColumn(int i, std::shared_ptr<ArrowType>& a)
+        : field_index(i), array(a) {}
+  };
+
+  template <template <typename> class Vector>
+  static Result<void> GetEntityTypeIDsFromProperties(
```

Comment:
This is a trivial comment, but functions that start with `Get `tend to be simple accessors.  This looks more like `RebuildEntityTypeIDsFromProperties`.

Reply:
I agree. Good suggestion.